.do/k8s-cluster/Makefile

```diff
@@ -6,7 +6,7 @@ TF ?= $(TF_EXTRA_OPTS) terraform
 WORKSPACE ?= ${DO_TF_WORKSPACE}
 VARS_FILE ?= deploy.tfvars
 NGINX_INGRESS_VERSION ?= 2.16.0
-CERT_MANAGER_VERSION ?= v1.0.1
```

Comment:
This version was having issues with the cert-manager-webhook pod which would repeatedly crash with issues reaching letsencrypt